<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1335</width>
    <height>767</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Main Window</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../Pics/57c3cc60caf48156d4d65a3f.png</normaloff>../../Pics/57c3cc60caf48156d4d65a3f.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="tableView_Movies">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelMainTable">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Movie table</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxMovieControl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>370</y>
      <width>381</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Movie library control</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_MovieTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>361</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labelMovieTitle">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Title</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_MovieAuthor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>361</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_MovieAuthor">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Author</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_RentPrice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_RentPrice">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Rent price (daily)</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addNewMovie">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add new movie</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_movieUpdateSelected">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>181</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Update selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_movieDeleteSelected">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>190</y>
       <width>171</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete selected</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LoadMovieLibrary">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>182</x>
       <y>0</y>
       <width>191</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Load movie library</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_Customers">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Customers table</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_CustomersControl">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>370</y>
      <width>381</width>
      <height>331</height>
     </rect>
    </property>
    <property name="title">
     <string>Customers control</string>
    </property>
    <widget class="QPushButton" name="pushButton_LoadCustomers">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>0</y>
       <width>191</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Load customers</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CustomerName">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>351</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_CustomerAdress">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>351</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PhoneNumber">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>171</width>
       <height>26</height>
      </rect>
     </property>
     <property name="inputMethodHints">
      <set>Qt::ImhDigitsOnly</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Adress</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Phone number</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_AddNewCustomer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>351</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Add new customer</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_UpdateCustomer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>171</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Update info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_DeleteCustomer">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>290</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete customer</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableView" name="tableView_Payments">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>30</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>10</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Payments table</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>230</y>
      <width>461</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Payments control</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_PaymentMethod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>441</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PaymentAmount">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PaymentDeposit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>113</width>
       <height>26</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Payment method</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Amount</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Deposit</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_isPaymentCompleted">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>161</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Payment completed</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_addNewPayment">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>138</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Add new payment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_updatePaymentInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>151</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Update info</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_deletePayment">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete payment</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_LoadPayments">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>0</y>
       <width>161</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>Load payments</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>550</y>
      <width>461</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Orders</string>
    </property>
    <widget class="QPushButton" name="pushButton_openOrdersReturnsWindow">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>141</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Create a new order</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_openReturnWindow">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>20</y>
       <width>131</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Return an order</string>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>20</y>
       <width>141</width>
       <height>26</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>12</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>5</month>
       <day>31</day>
      </datetime>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>50</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Today is:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_setCurrentDate">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>50</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>271</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Queries</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>610</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Connection</string>
    </property>
    <widget class="QPushButton" name="pushButton_connectToDb">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>361</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>CONNECT TO DATABASE</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ConnectionStatus">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>351</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Please connect to database.</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1335</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
